From: =?utf-8?q?Jeremy_B=C3=ADcha?= <jeremy.bicha@canonical.com>
Date: Tue, 15 Aug 2023 14:54:39 -0400
Subject: Revert "keyboard: Spawn "tecla" to show keyboard map"

Tecla is not ready yet:
https://gitlab.gnome.org/GNOME/tecla/-/issues/7

This reverts commit 4e340b76026ffe564941bcc7cfa8f7d2f09b2bc8
---
 panels/keyboard/cc-input-list-box.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/panels/keyboard/cc-input-list-box.c b/panels/keyboard/cc-input-list-box.c
index 225f6cf..2a0222c 100644
--- a/panels/keyboard/cc-input-list-box.c
+++ b/panels/keyboard/cc-input-list-box.c
@@ -239,10 +239,10 @@ row_layout_cb (CcInputListBox *self,
   layout_variant = cc_input_source_get_layout_variant (source);
 
   if (layout_variant && layout_variant[0])
-    commandline = g_strdup_printf ("tecla \"%s+%s\"",
+    commandline = g_strdup_printf ("gkbd-keyboard-display -l \"%s\t%s\"",
 				   layout, layout_variant);
   else
-    commandline = g_strdup_printf ("tecla %s",
+    commandline = g_strdup_printf ("gkbd-keyboard-display -l %s",
 				   layout);
 
   g_spawn_command_line_async (commandline, NULL);
